packages/react/popper/src/Popper.tsx

```diff
@@ -156,6 +160,30 @@ PopperArrow.displayName = ARROW_NAME;
 
 /* -----------------------------------------------------------------------------------------------*/
 
+const WINDOW_RESIZE_DEBOUNCE_WAIT_IN_MS = 100;
+
+function useWindowSize() {
+  const [windowSize, setWindowSize] = React.useState<Size | undefined>(undefined);
+
+  React.useEffect(() => {
+    let debounceTimerId: number;
+
+    function handleResize() {
+      window.clearTimeout(debounceTimerId);
+      debounceTimerId = window.setTimeout(
+        () => setWindowSize({ width: window.innerWidth, height: window.innerHeight }),
+        WINDOW_RESIZE_DEBOUNCE_WAIT_IN_MS
+      );
+    }
+
+    handleResize();
```

Comment:
minor but doesn't look like this one needs the delay? i'm guessing this is to return the initial dims?

Reply:
Done